modules/social_features/social_group/modules/social_group_request/config/install/message.template.request_to_join_a_group.yml

```diff
@@ -0,0 +1,37 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - activity_logger
+third_party_settings:
+  activity_logger:
+    activity_bundle_entities:
+      group_content-group_content_type_7fcb76fdf61a9: group_content-group_content_type_7fcb76fdf61a9
+      group_content-group_content_type_c8bae2527294c: group_content-group_content_type_c8bae2527294c
+    activity_action: create_entitiy_action
+    activity_context: group_content_in_my_group_activity_context
+    activity_destinations:
+      notifications: notifications
+      email: email
+    activity_create_direct: true
+    activity_aggregate: false
+    activity_entity_condition: ''
+template: request_to_join_a_group
+label: 'Request to join a group'
+description: 'A person requests membership to a group I manage'
+text:
+  -
+    value: "<p><a href=\"[message:recipient-user:url:absolute]\">[message:recipient-user:display-name]</a> has requested membership to the&nbsp;<a href=\"[message:gurl]\">[message:gtitle]</a> group</p>\r\n"
```

Comment:
Same here, could you remove the `&nbsp;` from the values below?

Reply:
Removed